src/net.cpp

```diff
@@ -1187,7 +1204,11 @@ void CConnman::AcceptConnection(const ListenSocket& hListenSocket) {
     pnode->fWhitelisted = whitelisted;
     GetNodeSignals().InitializeNode(pnode, *this);
 
-    LogPrint(BCLog::NET, "connection from %s accepted\n", addr.ToString());
+    if (fLogIPs) {
+        LogPrint(BCLog::NET, "connection from %s accepted\n", addr.ToString());
+    } else {
+        LogPrint(BCLog::NET, "connection accepted\n", addr.ToString());
```

Comment:
I don't think your 3rd argument is necessary here.

Reply:
ooops 🙈